XTS/raw-xts-api-demo/test/pom.xml

```diff
@@ -144,13 +150,15 @@
         <profile>
             <id>arq-wildfly-managed</id>
             <activation>
-                <activeByDefault>true</activeByDefault>
+                <property>
+                    <name>!no.arq</name>
+                </property>
```

Comment:
I still think that this is not the right way to activate a profile in our quickstarts. I would prefer to require users to activate their preferred arquillian profile. Having said that, how does it work when users want to activate `arq-wildfly-remote` instead of `arq-wildfly-managed`? Should they specify `-Dno.arq -Parq-wildfly-remote`?

Reply:
Thanks @jmfinelli for this comment!! This is my mistake! Seeing that jdk17 is not needed anymore in XTS I am going to restore the original status of profile activation. I agree that users should be able to activate their preferred arquillian profile as we discussed.

Reply:
I think that we can do something better here (as we discussed with Tom) but if you prefer not to modify this part in this PR, I am alright with that as well ;-)

Reply:
I agree with you Manuel, but I would prefer to create a specific issue for that because this PR. Would you please see this issue I have just created (https://issues.redhat.com/browse/JBTM-3686) and add some additional description (if needed) ?